web/modules/custom/ys_base/tests/src/Functional/YsBaseFunctionalTestBase.php

```
@@ -0,0 +1,32 @@
+ */
+abstract class YsBaseFunctionalTestBase extends BrowserTestBase {
+
+  use ArrayTrait;
+  use AssertTrait;
+  use MockTrait;
+  use ReflectionTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
```

PHPStan (2.1.15)

Property Drupal\Tests\ys_base\Functional\YsBaseFunctionalTestBase::$defaultTheme has no type specified.


(missingType.property)